docs/enterprise.md

```diff
@@ -0,0 +1,104 @@
+# Logfire Enterprise
+
+## Overview
+
+In addition to the [Pro plan](https://pydantic.dev/pricing), Logfire has two enterprise offerings:
+
+- **Enterprise Cloud**: Fully managed, SLA-backed service
+- **Enterprise Self-Hosted**: On-premises deployment via Kubernetes
+
+> **Note:** The Helm chart for Pydantic Logfire is now [open source](https://github.com/pydantic/logfire-helm-chart)

```

Comment:
I don't know what helm charts are so when I read this I don't know which of the two options above this is related to or why I'm being told this. I think remove this line and convert the mention in the table below to a link.

Reply:
moved to self-hosted section